package.json

```diff
@@ -54,6 +54,8 @@
     "istanbul": "~0.1.34",
     "sinon-chai": "2.4.0",
     "timekeeper": "0.0.5",
+    "nock": "9.0.14",
```

Comment:
Note that nock is included twice... I think is a consecuence of last merge from master

Reply:
Fixed in fdd0abfc236dfdb391572e67b1e675785f3c4ce1

Reply:
Fixed in fdd0abfc236dfdb391572e67b1e675785f3c4ce1